### PR TITLE
Adding ThirdPartyNotices.txt

### PR DESCRIPTION
**Pull request by Dmitry-Matveev**

### CODE REVIEW

File: ThirdPartyNotices.txt [hunk 1]
```diff
@@ -0,0 +1,607 @@
+﻿THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+Microsoft Application Insights Java Software Development Kit consists of material from the project(s) listed below (collectively, �Distributed Code�). Microsoft Corporation ("Microsoft") is not the original author of some of the Distributed Code. The original copyright notices and licenses, under which Microsoft received such Distributed Code, are set out below.  This Distributed Code are licensed to you under their original license terms set forth below.  Microsoft reserves all other rights not expressly granted under these terms, whether by implication, estoppel or otherwise.
```
Comment (SergeyKanzhelev) [inline ThirdPartyNotices.txt]: fix �

File: ThirdPartyNotices.txt [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+9. Accepting Warranty or Additional Liability. While redistributing the Work or Derivative Works thereof, You may choose to offer, and charge a fee for, acceptance of support, warranty, indemnity, or other liability obligations and/or rights consistent with this License. However, in accepting such obligations, You may act only on Your own behalf and on Your sole responsibility, not on behalf of any other Contributor, and only if You agree to indemnify, defend, and hold each Contributor harmless for any liability incurred by, or claims asserted against, such Contributor by reason of your accepting any such warranty or additional liability.
+=========================================
+END OF Apache Commons-codec NOTICES, INFORMATION, AND LICENSE
+
+%% Apache Commons Lang3 NOTICES, INFORMATION, AND LICENSE BEGIN HERE
+=========================================
+Copyright � 2001-2014 The Apache Software Foundation
```
Comment (SergeyKanzhelev) [inline ThirdPartyNotices.txt]: �

File: ThirdPartyNotices.txt [hunk 3]
```diff
@@ -0,0 +1,607 @@
...
+9. Accepting Warranty or Additional Liability. While redistributing the Work or Derivative Works thereof, You may choose to offer, and charge a fee for, acceptance of support, warranty, indemnity, or other liability obligations and/or rights consistent with this License. However, in accepting such obligations, You may act only on Your own behalf and on Your sole responsibility, not on behalf of any other Contributor, and only if You agree to indemnify, defend, and hold each Contributor harmless for any liability incurred by, or claims asserted against, such Contributor by reason of your accepting any such warranty or additional liability.
+=========================================
+END OF Apache Commons-codec NOTICES, INFORMATION, AND LICENSE
+
+%% Apache Commons Lang3 NOTICES, INFORMATION, AND LICENSE BEGIN HERE
```
Comment (SergeyKanzhelev) [inline ThirdPartyNotices.txt]: what is %% for?

Comment (Dmitry-Matveev) [inline ThirdPartyNotices.txt]: It was a separator in the original file formatting, not sure why exactly %% was selected...

File: ThirdPartyNotices.txt (hunk 2 above)
Comment (Dmitry-Matveev) [inline ThirdPartyNotices.txt]: Replaced with (c).
The other places with Copyright were not marked with (c) in the original version of the document, so I kept them as they were...

File: ThirdPartyNotices.txt (hunk 1 above)
Comment (Dmitry-Matveev) [inline ThirdPartyNotices.txt]: Thanks, done!

**Pull request merged**